doc/src/reference.xml

```diff
@@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
       <varlistentry>
         <term><literal>cflags</literal></term>
         <term><literal>cxxflags</literal></term>
+        <term><literal>compileflags</literal></term>
```

Comment:
compileflags is not a feature.  What you're thinking of is the use of compileflags here: http://www.boost.org/build/doc/html/bbv2/reference/tools.html, which is different.  I realize that this is confusing, but the existing documentation does describe the current behavior correctly.